src/code/z_demo.c

```diff
@@ -264,107 +296,131 @@ void func_80064824(PlayState* play, CutsceneContext* csCtx, CsCmdBase* cmd) {
-                    csCtx->state = CS_STATE_UNSKIPPABLE_INIT;
+
+        case CS_MISC_STOP_CUTSCENE:
+            if (isFirstFrame) {
+                if (csCtx->state != CS_STATE_RUN_UNSTOPPABLE) {
+                    csCtx->state = CS_STATE_STOP;
                 }
             }
             break;
-        case 13:
+
+        case CS_MISC_TRIFORCE_FLASH:
             if (play->roomCtx.unk_74[1] == 0) {
                 func_80078884(NA_SE_EV_TRIFORCE_FLASH);
             }
-            if (play->roomCtx.unk_74[1] < 0xFF) {
+
+            if (play->roomCtx.unk_74[1] < 255) {
                 play->roomCtx.unk_74[1] += 5;
             }
             break;
-        case 14:
-            if (sp3F != 0) {
+
+        case CS_MISC_SET_LOCKED_VIEWPOINT:
+            if (isFirstFrame) {
                 Play_SetViewpoint(play, VIEWPOINT_LOCKED);
             }
             break;
-        case 15:
-            if (sp3F != 0) {
+
+        case CS_MISC_SHOW_TITLE_CARD:
+            if (isFirstFrame) {
                 TitleCard_InitPlaceName(play, &play->actorCtx.titleCtx, player->giObjectSegment, 160, 120, 144, 24, 20);
             }
             break;
-        case 16:
-            if (sp3F != 0) {
+
+        case CS_MISC_QUAKE_START:
+            if (isFirstFrame) {
                 sQuakeIndex = Quake_Request(GET_ACTIVE_CAM(play), QUAKE_TYPE_6);
                 Quake_SetSpeed(sQuakeIndex, 0x7FFF);
                 Quake_SetPerturbations(sQuakeIndex, 4, 0, 1000, 0);
                 Quake_SetDuration(sQuakeIndex, 800);
             }
             break;
-        case 17:
-            if (sp3F != 0) {
+
+        case CS_MISC_QUAKE_STOP:
+            if (isFirstFrame) {
                 Quake_RemoveRequest(sQuakeIndex);
             }
             break;
-        case 18:
+
+        case CS_MISC_STOP_STORM_AND_ADVANCE_TO_DAY:
             play->envCtx.precipitation[PRECIP_RAIN_MAX] = 0;
             play->envCtx.stormRequest = STORM_REQUEST_STOP;
+
             if (gSaveContext.dayTime < CLOCK_TIME(7, 0)) {
                 gSaveContext.dayTime += 30;
             }
+
             if (play->envCtx.precipitation[PRECIP_RAIN_CUR] == 0) {
                 gWeatherMode = WEATHER_MODE_CLEAR;
                 Audio_SetNatureAmbienceChannelIO(NATURE_CHANNEL_RAIN, CHANNEL_IO_PORT_1, 0);
             }
             break;
-        case 19:
+
+        case CS_MISC_SET_FLAG_FAST_WINDMILL:
             SET_EVENTCHKINF(EVENTCHKINF_65);
             break;
-        case 20:
+
+        case CS_MISC_SET_FLAG_WELL_DRAINED:
             SET_EVENTCHKINF(EVENTCHKINF_67);
             break;
-        case 21:
+
+        case CS_MISC_SET_FLAG_LAKE_HYLIA_RESTORED:
             SET_EVENTCHKINF(EVENTCHKINF_69);
             break;
-        case 22:
-            D_801614B0.r = 255;
-            D_801614B0.g = 255;
-            D_801614B0.b = 255;
-            D_801614B0.a = 255;
+
+        case CS_MISC_VISMONO_BLACK_AND_WHITE:
```

Comment:
"gray"/"grayscale" would be more accurate

Reply:
black and white is how literally anyone would refer to this state, like "black and white tvs"

Reply:
but it's not just black and white there is a whole range of gray in between <img src="https://cdn.discordapp.com/emojis/1007556815820689508.webp?size=128&quality=lossless" alt="PepeHands" height="32">

I agree "black and white" will be understood as "grayscale" by everyone, I just wish people used "grayscale" instead <img src="https://cdn.discordapp.com/emojis/982515528096682014.webp?size=128&quality=lossless" alt="Copium" height="24">